File: javascript/math/CountDigit.js
class CountDigit {
    countDigit(number) {
        var count = 0;
        while (number != 0) {
            count++;
            number = Math.floor(number / 10);
        }
        return count;
    }
}

console.log(new CountDigit().countDigit(1223));
console.log(new CountDigit().countDigit(122354));
console.log(new CountDigit().countDigit(1));

File: python/educational/CountDigit.py
def count_digit(value):
    """Count the number of digits in the number passed into this function"""
    digit_counter = 0
    while value > 0:
        digit_counter = digit_counter + 1
        value = value // 10
    return digit_counter


print("Length of the number 1111 = ", count_digit(1111))